docs/source/cuda.rst

```diff
@@ -3,68 +3,104 @@ torch.cuda
 
 .. currentmodule:: torch.cuda
 
-.. automodule:: torch.cuda
-   :members:
+.. autosummary::
```

Comment:
Hmm, we lost the nice description:

Before:
![image](https://user-images.githubusercontent.com/5652049/114428132-d0276680-9b89-11eb-8b8b-6a5ca3e8f2fc.png)

After:
![image](https://user-images.githubusercontent.com/5652049/114428113-cbfb4900-9b89-11eb-81d9-98becbb0a2ec.png)

Reply:
restored